mava/components/jax/building/adders.py

```diff
@@ -152,7 +178,7 @@ def on_building_executor_adder(self, builder: SystemBuilder) -> None:
 
         adder = reverb_adders.ParallelNStepTransitionAdder(
             priority_fns=builder.store.priority_fns,
-            client=builder.store.data_server_client,
+            client=builder.store.data_server_client,  # Created by builder
```

Comment:
Can probably remove this comment?

Reply:
I included these comments in a bunch of places to mark where store variables are created by core systems, instead of by components. Do you not find them to be useful when reading and trying to understand the code?

Reply:
I tihnk they are useful in that case. Please ignore my comments regarding the comments :sweat_smile: